config/routes/api/v2.rb

```diff
@@ -304,6 +304,13 @@ class ActionDispatch::Routing::Mapper
           end
         end
 
+        api_resources :traces, :only => [], :controller => 'host_tracer' do
+          collection do
+            match '/traces' => 'traces#resolve_traces', :via => :get
```

Comment:
I think this GET route can be removed. The API should be using the PUT route

Reply:
Fixed